src/main/java/hu/dpc/rt/kafkastreamer/importer/ElasticsearchTemplateSetup.java

```diff
@@ -39,42 +39,15 @@ private void createIndexTemplates() {
         if (indexConfiguration.createTemplate) {
             createRootIndexTemplate();
 
-            if (indexConfiguration.deployment) {
```

Comment:
You can't remove unnecessary (at the moment) indexes like this. This code is still valuable. We want to pass it as a configuration in application.yml just like zeebe elasticsearch exporter.

Reply:
I am trying to do the same.